src/state/trajectory/reducer.ts

```diff
@@ -73,6 +75,13 @@ const actionToConfigMap: TypeToDescriptionMap = {
             simulariumFile: initialState.simulariumFile,
         }),
     },
```

Comment:
At this point the action does not replace anything in state and is just a vehicle to run the logic.